src/utilities/ShortcutManager.ts

```diff
@@ -0,0 +1,119 @@
+const MODIFIER_KEY_CODES = ['ControlLeft', 'ControlRight', 'ShiftLeft', 'ShiftRight', 'AltLeft']
+
```

Comment:
Sure, I’ll do that.

My rational here is that I try to name files after the primary/sole export of the module because it aids me in tasks like project-wide text search. Searching for `ShortcutManager` will now yield in-file references but also references to the file path. It’s a minor thing though.

Reply:
I didn't realize this was exporting a class or I wouldn't have made the suggestion 😬 

I'm going to go ahead and approve as-is to unblock, but if you'd like to change back to PascalCase you're more than welcome.